lib/cpp/src/thrift/concurrency/PosixThreadFactory.h

```diff
@@ -39,7 +39,14 @@ class PosixThreadFactory : public ThreadFactory {
   /**
    * POSIX Thread scheduler policies
    */
-  enum POLICY { OTHER, FIFO, ROUND_ROBIN };
+  class Policy {
+  public:
+    enum value {
+      OTHER       = 0,
+      FIFO        = 1,
+      ROUND_ROBIN = 2
+    };
```

Comment:
Intention was to make it more readable.

In code, OTHER, FIFO, ROUND_ROBIN mixed with NORMAL, LOWER, HIGHER... this adds a scope to them, i.e. "Policy::FIFO" and "Priority::NORMAL".

I can revert it if needed, I just wanted to make the code more maintainable.  I suspect these options are not used that often anyway.